### ISSUE TITLE
The GUI is a mess! 

### ISSUE THREAD

**Issue by TateA** [anbox/anbox#876]
**System Specs:**
**OS:** Ubuntu Desktop 16.04.5 LTS 64 Bit
**CPU:** AMD Ryzen 7 1700
**GPUs:** AMD Radeon RX 580 8GB, AMD Radeon RX 570 8GB, and AMD Radeon RX 570 8GB ( amdgpu-pro-17.40-483984 )

Anbox was installed from PPA, installation instructions: https://github.com/anbox/anbox/blob/master/docs/install.md

Heres the out put of `sudo /snap/bin/anbox.collect-bug-info`: [anbox-system-diagnostics-2018-08-22.zip](https://github.com/anbox/anbox/files/2312311/anbox-system-diagnostics-2018-08-22.zip)

**Here the out put for version: **
```
$ anbox version
anbox local-7cc6fb4
```

When I start *Anbox Application manager* from my Application launcher thingy on Ubuntu. It starts up, but the GUI is a f****** mess! 

![screenshot from 2018-08-22 19-30-22](https://user-images.githubusercontent.com/12721303/44496289-d87dac80-a641-11e8-9020-d66fcd3dfb49.png) 

If I click around in that mess I can open another window, that does not look much better.

![screenshot from 2018-08-22 19-35-10](https://user-images.githubusercontent.com/12721303/44496434-838e6600-a642-11e8-9ea1-f08068bd68c3.png)

**Comment by morphis** [comment]
@TateA I agree that the UI looks bad. However from the logs I couldn't figure a good reason. Can you stop the session manager and then start it manually with `ANBOX_LOG_LEVEL=debug anbox session-manager`. Once it said it booted please run `anbox.appmgr` for example and once the window is up and showing the crippled graphical output, please paste all output from the commands here.

**Comment by TateA** [comment]
Okay I reinstalled Ubuntu, but with out AMDGPU Pro 18.30 divers. And it seems to work!! :)

![image](https://user-images.githubusercontent.com/12721303/45589427-42147200-b8f3-11e8-9435-7dcadbe58036.png)

Will let you guys know if i run into any problems, lol. ;)
